_sass/components/_navbar.scss

```diff
@@ -4,18 +4,20 @@
   color: $color-primary-darkest;
   display: flex;
   flex-shrink: 0;
-  font-size: $h5-font-size;
+  font-size: 21px;
+  font-stretch: ultra-condensed;
   font-weight: $font-bold;
+  letter-spacing: 0.1px;
   text-decoration: none;
   width: 15rem;
 
   @media (min-width: $width-sm) {
-    font-size: $h3-font-size;
+    // font-size: 12px;
```

Comment:
if we don't need this, can we delete it?

Reply:
Yes we can

Reply:
Removed!